tests/training/test_autotrain.py

```diff
@@ -0,0 +1,77 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import os
+
+import pytest
+from argilla.training import ArgillaTrainer
+
+FRAMEWORK = "autotrain"
+MODELS = ["prajjwal1/bert-tiny", "autotrain"]
+_HF_HUB_ACCESS_TOKEN = os.environ.get("HF_AUTH_TOKEN") or os.environ.get("HF_HUB_ACCESS_TOKEN")
```

Comment:
Here indeed we could mock it as I did for the `push_to_huggingface` and `from_huggingface` methods in `FeedbackDataset`, but we can have a look at this later together 😄

Reply:
I agree!